libnavigation-ui/src/main/res/values/styles.xml

```diff
@@ -75,10 +75,56 @@
 
         <item name="navigationViewMapStyle">@string/navigation_guidance_night</item>
     </style>
-    
+
     <style name="NavigationLocationLayerStyle" parent="@style/mapbox_LocationComponent">
         <item name="mapbox_trackingAnimationDurationMultiplier">2.0</item>
         <item name="mapbox_trackingGesturesManagement">true</item>
     </style>
 
+    <!-- Default InstructionView style -->
+    <style name="InstructionView">
```

Comment:
after this change, `NavigationView` is also a consumer of each UI component. It will also need to define the style for each UI component.